.circleci/config.yml

```diff
@@ -169,13 +245,42 @@ workflows:
           filters:
             branches:
               ignore: main
+      - bump-dev-version:
+          filters:
+            branches:
+              only:
+                - develop
+                - /.*-publish-dev-package/
+          requires:
+            - pip-install
+      - build-package:
+          name: build-dev-package
+          requires:
+            # - webpack
+            # - compile-translations
```

Comment:
I wasn't sure about these two lines. As far as I understand these lines only apply to integreat-cms and not to the compass, right?

Reply:
Since we also use webpack and translations for this project, we also need the corresponding jobs here.
We don't want webpack to be running on the production server, thus we need one step to bundle the static files and put them in the python package as well.
Same with the translations, since we don't have the compiled translations in our repository, we need to compile them first and pass them to the packaging job in order for them to be present in the resulting package.

Reply:
Ok, thank you for the explanation. It's still going to take a minute to wrap my head around it, but this helps for now. Is it enough for this PR comment the two jobs in again or is it also part of this PR to define the webpack job [like done here](https://github.com/digitalfabrik/integreat-cms/blob/develop/.circleci/config.yml#L197-L211)?

Reply:
We have to add the job definition here, otherwise the workflow will just fail because the required jobs do not exist.

Reply:
The requested change should be implied